tastytrade/instruments.py

```diff
@@ -1,21 +1,24 @@
 from dataclasses import dataclass
 from datetime import date, datetime
-from typing import Any, Optional
+from typing import Any, Optional, TypedDict
 
 import requests
 
 from tastytrade.session import Session
 from tastytrade.utils import snakeify, validate_response
 
-
-@dataclass
-class DestinationVenueSymbol:
-    id: int
-    symbol: str
-    destination_venue: str
-    routable: bool
-    max_quantity_precision: Optional[int] = None
-    max_price_precision: Optional[int] = None
+DestinationVenueSymbol = TypedDict(
+    'DestinationVenueSymbol',
+    {
+        'id': int,
+        'symbol': str,
+        'destination_venue': str,
```

Comment:
this is with dashes now not underscores; remember, this isn't a real type a runtime, it's just there for the user's convenience